keyboards/massdrop/alt/keymaps/pregame/keymap.c

```diff
@@ -0,0 +1,193 @@
+        _______, _______, _______,                            _______,                            _______, _______, CHART,   CHART,   CHART,
+        //UnderGlow
+        CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART
+        },
+    [1] = {
+        CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,    CHART,   CHART,   CHART,   CHART,     CHART,    BLUE,
+        GOLD,    PINK,    AZURE,   PINK,    TURQ,    TEAL,    _______, RED,     _______,  _______, GREEN,   BLUE,    GOLD,      _______,  BLUE,
+        TEAL,    MAGENT,  AZURE,   MAGENT,  TURQ,    TEAL,    _______, _______, _______,  _______, _______, _______,                GOLD, BLUE,
+        _______, _______, _______, _______, _______, RED,     RED,     RED,     _______,  _______, _______, _______,                GOLD, BLUE,
+        _______, _______, _______,                            GOLD,                                _______, _______,       PURPLE,  GOLD, PURPLE,
+        //UnderGlow
+        CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,                                                                                                                        CHART,
+        CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART,   CHART
+    },
+};
+
+#undef _______
+#define _______ KC_TRNS
+#endif
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+    // Enable or disable debugging
+    debug_enable=true;
+    debug_matrix=true;
+    debug_keyboard=true;
```

Comment:
Do you really want to enable debugging all the time?  
Maybe put wrap them inside an ifdef. (or just disable them)